Tools/WinMLDashboard/public/convert.py

```diff
@@ -87,7 +87,7 @@ def libSVM_converter(args):
                                  target_opset=get_opset(args.ONNXVersion))
     return onnx_model
 
-def convert_tensorflow_file(filename, opset, input_names, output_names):
+def convert_tensorflow_file(filename, opset, output_names):
```

Comment:
Should line 100 be `converted_model = onnxmltools.convert_tensorflow(..., input_names=input_names, ...)` ?

Reply:
The input names aren't needed for calling into the tensorflow conversion. In this previous PR (https://github.com/microsoft/Windows-Machine-Learning/pull/361/files) I mistakenly added this parameter to the convert_tensorflow_file function, and so I removed it now.